930-delegate-tasks/src/main/java/software/wings/delegatetasks/jira/JiraTask.java

```
@@ -891,7 +892,11 @@ protected DelegateResponseData checkJiraApproval(JiraTaskParameters parameters)
           return JiraExecutionData.builder().executionStatus(ExecutionStatus.FAILED).errorMessage(errorMessage).build();
         }
       }
-      log.error(errorMessage, e);
+      if (isWarningException(e)) {
+        log.warn(errorMessage);
+      } else {
+        log.error(errorMessage, e);
+      }
```

why not specifically catch SSL exception and log.warn that?

---

It comes inside the original exception. To follow that way we need to re-throw the inner exception and that is an unnecessary overhead.

---

got it, thanks!